### ISSUE TITLE
New place on the map 🗺 

### ISSUE THREAD

**Issue by stonejess** [stonejess/ideas#3]
I was thinking we should add a extension of eroa. Once you beat the boss, you enter the mining shaft and you are in a cave. I have bats 🦇Miner A, Cursed gem B,

**Comment by stonejess** [comment]
And the background ready if we are going to add it to the app

**Comment by stonejess** [comment]
Other people have been added since last post

**Comment by ghost** [comment]
Yes that's perfect! But the background that's ready, is that the one that @vishwamr made? He already made one. But yes, I like your idea about having it in an extension to Eora. That's what we should do.

Also, could you put these in the b-Ocracy repo (this repo could have ideas for future games).

Thanks!

**Comment by stonejess** [comment]
Both, mine and his

**Comment by VishwamR** [comment]
The one I made it's a cave for the boss not the regular background

>

**Comment by ghost** [comment]
Okay so we'll have a cave near Eora that will have @stonejess's background for them and @VishwamR's background will be for the boss battles. @ethanchoo, does this sound good?
